src/edu/cmu/cs/sasylf/ast/CompUnit.java

```diff
@@ -224,11 +238,15 @@ public void collectQualNames(Consumer<QualName> consumer) {
 
 	@Override
 	public Object getDeclaration(Context ctx, String name) {
-		if (cacheVersion != ctx.version()) {
+		// TODO: We need to figure out how to handle the cache properly
+		if (true || cacheVersion != ctx.version()) {

```

Comment:
Caching isn't done frequently enough, so I wrote `true ||` here to not use the cached data. I am still working on a way around this.

Reply:
> Yes, currently, there are regression tests that fail if we get rid of `true ||`. I'm still trying to figure why this is the case, but, for now, having `true ||` eliminates the bug.

Reply:
Fixed